views/jsconfig.handlebars

```diff
@@ -5,11 +5,24 @@ if (isIE && !isUnsupportedPage) {
   window.location.replace('/unsupported/ie');
 }
 
-{{#if dsn}}
-window.dsn = '{{{dsn}}}';
+{{#if sentryId}}
+var RAVEN_CONFIG = {
+  release: '{{{version}}}',
+  tags: {
+    commit: '{{{commit}}}'
+  },
+  dataCallback: function (data) {
+    var hash = window.location.hash;
+    if (hash) {
+      return JSON.parse(JSON.stringify(data).replace(new Regexp(hash.slice(1), 'g'), ''));
```

Comment:
I think throwing is better in this case. It has the least surprising result, the report isn't sent. Returning `null` sends the original `data` including fragment, and returning `{}` also doesn't send the report but is less obvious what it might do.